xonsh/environ.py

```diff
@@ -1663,9 +1661,7 @@ def __str__(self):
         return str(self._d)
 
     def __repr__(self):
-        return "{0}.{1}(...)".format(
-            self.__class__.__module__, self.__class__.__name__, self._d
-        )
+        return "{0}.{1}(...)".format(self.__class__.__module__, self.__class__.__name__)
```

Comment:
Same as above: Why did you change the `__repr__` here? Does this belong to this patchset?

Reply:
*idem*